docgen/src/guide/Migration_guide_v5.md

```diff
@@ -10,15 +10,15 @@ React V5 introduces **a complete revamp of the HTML output of most widgets**. Th
 
 This release also introduces a **new CSS naming convention** which will be reused across all InstantSearch libraries. This will enable the possibility to develop cross-libraries CSS themes easily.
 
-This guide will provide step-by-step migration information for each React InstantSearch widget.
+This guide will provide step-by-step migration information for each widget & connector.
 
 ## Table of contents
 
 * [Migration steps](guide/Migration_guide_v5.html#migration-steps)
   * [Updating widgets names](guide/Migration_guide_v5.html#updating-widgets-names)
-  * [Updating Panel](guide/Migration_guide_v5.html#updating-panel)
-  * [autoHideContainer](guide/Migration_guide_v5.html#autohidecontainer)
+  * [Updating props names](guide/Migration_guide_v5.html#updating-props-names)
```

Comment:
Nope since we are using `<base href="/">` the links will forward the page to `/#xxx`.

https://github.com/algolia/react-instantsearch/blob/205807166a25065c57287f218055d1904e18b9b5/docgen/layouts/common/meta.pug#L12